crypto/evp/evp_lib.c

```diff
@@ -331,6 +347,10 @@ unsigned long EVP_CIPHER_flags(const EVP_CIPHER *cipher)
     params[0] = OSSL_PARAM_construct_ulong(OSSL_CIPHER_PARAM_FLAGS, &v);
     ok = evp_do_ciph_getparams(cipher, params);
 
+    /* Provided implementations may have a custom cipher_cipher */
+    if (cipher->prov != NULL && cipher->ccipher != NULL)
+        v |= EVP_CIPH_FLAG_CUSTOM_CIPHER;
```

Comment:
I dont know that this is a good idea at the moment... Maybe this PR should just deal with the return 0 issues..

Reply:
There's no real way around it unless you want to mess around with that flag in the implementations. After all, an existing ccipher that behaves like cupdate should be harmless, and because the returned value from EVP_Cipher is tied to that flag...